blockmap.js

```diff
@@ -34,13 +34,13 @@ function mergeBlocks(blocks) {
 }
 
 function streamSha256(stream, callback) {
-	const hash = sha256.create();
-	stream.on('data', function(data) {
-		hash.update(data);
-	});
+	const hash = crypto.createHash('sha256');
+	hash.setEncoding('hex');
```

Comment:
I don't think this is crypto's API. https://nodejs.org/api/crypto.html#crypto_crypto_createhash_algorithm